viper

```diff
@@ -38,11 +38,13 @@ start () {
     volume=$( pactl list sinks | grep '^[[:space:]]Volume:' | head -n "$sink" | tail -n 1 | sed -e 's,.* \([0-9][0-9]*\)%.*,\1,' )
     echo $location > $originalsinkfile
     echo $volume > $originalvolumefile
+    [ "$volume" ] || vvol="100"
     
 	echo "Setting original sink to full volume..."
 	pactl set-sink-volume $location 1.0
 	echo "Changing primary sink to Viper..."
 	pactl set-default-sink $vipersink
+	pactl set-sink-volume $vipersink "${vvol}%"
```

Comment:
I seem to always have this error `Invalid volume specification` caused by `vvol` not being set. Is this line supposed to set `vvol` if `volume` is not set?

I changed the line to `if [[ -z "$volume" ]]; then vvol="100"; else vvol="$volume" ; fi` and it seems to work, though I'm not sure if that was your intended purpose.

Here is the full console log:
```
Killed process. Deleted pidfile.
Reset original sink volume.
Unloaded Viper sink.
Storing original sink data...
Setting original sink to full volume...
Changing primary sink to Viper...
Invalid volume specification
Moving existing audio streams to Viper...
```